APITesters/SwiftAPITester/SwiftAPITester/StoreProductAPI.swift

```diff
@@ -8,6 +8,8 @@
 import RevenueCat
 
 var product: StoreProduct!
+
+// swiftlint:disable:next function_body_length
```

Comment:
Maybe we could disable these for the APITester folders?

Reply:
How do we do that? 😇

Reply:
You could create a `.swiftlint.yml` (like `PurchasesTests/.swiftlint.yml`) inside of `APITester/` with this rule, then I think it'll disable it for all testers since it's not a rule we care about for these files :) But we can clean this up after this PR.

Reply:
Done: #1309